src/domains/User/UserProfile/UserProfile.tsx

```diff
@@ -19,37 +31,185 @@ const useStyles = makeStyles((theme) => ({
+                        <Avatar src={img} alt='Profile Avatar' />
+                    </Grid>
+                    
+                    <Grid component='span' item xs={12}>
+                        {/* <Form onSubmit={handleNameSubmit}> */}
+                        <Form onSubmit={handleSubmit(sendName)}>
+                            <FormContent>
+                                <TextField
+                                    inputProps={{ 'aria-label': 'First Name' }}
+                                    label='First Name'
+                                    aria-label='First Name'
+                                    id='fName'
+                                    required
+                                    type='text'
+                                    placeholder='Your First Name Here'
+                                    value={regForm.firstName}
+                                    // value={user.name.first}
+                                    onChange={handleChange("firstName")}
+                                />
+                            </FormContent>
+                        </Form>
+                    </Grid>
+                    
+                    <Grid component='span' item xs={12}>
+                        {/* <Form onSubmit={handleNameSubmit}> */}
+                        <Form onSubmit={handleSubmit(sendName)}>
+                            <FormContent>
+                                <TextField
+                                    inputProps={{ 'aria-label': 'Last Name' }}
+                                    label='Last Name'
+                                    aria-label='Last Name'
+                                    id='fName'
+                                    required
+                                    type='text'
+                                    placeholder='Your Last Name Here'
+                                    value={regForm.lastName}
+                                    // value={user.name.last}
+                                    onChange={handleChange("lastName")}
+                                />
+                            </FormContent>
+                        </Form>
+                    </Grid>
+
+                    <Grid component='span' item xs={12}>                        
+                        {/* <Form onSubmit={handleEmailSubmit}> */}
+                        <Form onSubmit={handleSubmit(sendEmail)}>
+                            <FormContent>
+                                <TextField
+                                    inputProps={{ 'aria-label': 'E-mail' }}
+                                    label='Email'
+                                    aria-label='E-mail'
+                                    required
+                                    type='email'
+                                    placeholder='Your E-mail Here'
+                                    // does not properly validate emails
```

Comment:
Were you unable to get proper html5 email validation working?

Reply:
It was not working properly, as long as there was an `@` it validated so I reused the regex in the api to fix this.